### PR TITLE
tx-manager - add eip155 support

### PR DESCRIPTION
**Pull request by kumavis**

### CODE REVIEW

**Comment by kumavis** [comment]
also includes some improvement to the tx-manager-tests

File: test/unit/tx-manager-test.js [hunk 1]
```diff
@@ -1,19 +1,30 @@
-const assert = require('assert')
+// const assert = require('assert')
```
Comment (frankiebee) [inline test/unit/tx-manager-test.js]: remove this

**Review by frankiebee — APPROVED** [review]
Looks good to me

**Comment by kumavis** [comment]
This was QA'd on mainnet, morden, ropsten, and testrpc

File: test/unit/tx-utils-test.js [hunk 2]
```diff
@@ -12,6 +12,23 @@ describe('txUtils', function() {
...
+        from: '0x69ad465e0bab6504002ad58c744ed89c7da38525',
+        value: '0x0',
+        gas: '0x7b0c',
+        gasPrice: '0x199c82cc00',
+        data: '0x',
+        nonce: '0x3',
+        chainId: 42,
+      }
+      const ethTx = txUtils.buildEthTxFromParams(txParams)
+      assert.equal(ethTx.getChainId(), 42, 'chainId is set from tx params')
```
Comment (flyswatter) [inline test/unit/tx-utils-test.js]: I scrolled to the bottom and finally found this, the assertion that really proves the feature.  Sorry you had to adjust all those other tests to be compliant!

**Review by danfinlay — APPROVED** [review]
The code looks good, the tests seem to prove it works.

**Pull request merged**